### PR TITLE
Added HtmlToOpenXml to OpenXml section

### PR DESCRIPTION
**Pull request by onizet**

### CODE REVIEW

**Comment by onizet** [comment]
Ok thanks, I have updated my commit

**Comment by ngohungphuc** [comment]
@onizet please resolve conflict

**Comment by onizet** [comment]
I resolved the conflict then travis failed while scanning for site (not related with my PR). See the logs: 
> [L734]  https://www.dotnetrocks.com SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed 

What do you want I do?

File: README.md [hunk 1]
```diff
@@ -594,6 +594,8 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
 * [EntryPoint](https://github.com/Nick-Lucas/EntryPoint) - Composable CLI (Command Line) Argument Parser for .Net Core & .Net Framework 4.5+.
 * [Fake JSON Server](https://github.com/ttu/dotnet-fake-json-server) - Fake REST API for prototyping or as a CRUD Back End. No need to define types, uses dynamic typing. Data is stored to a single JSON file. Has authentication, WebSocket notifications, async long running operations, random generation for errors/delays and experimental GraphQL support.
 * [gitignore.io](https://github.com/joeblau/gitignore.io) - Create useful .gitignore files for your project [https://www.gitignore.io](https://www.gitignore.io).
+* [GitInfo](https://github.com/kzu/GitInfo) - Git and SemVer Info from MSBuild, C# and VB.
```
Comment (ngohungphuc) [inline README.md]: Hi @onizet I still see this line change can you please check it again. For the build fail if not related to your PR then is fine just ignore it. Thank you

File: README.md [hunk 2]
```diff
@@ -594,7 +594,6 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
 * [EntryPoint](https://github.com/Nick-Lucas/EntryPoint) - Composable CLI (Command Line) Argument Parser for .Net Core & .Net Framework 4.5+.
 * [Fake JSON Server](https://github.com/ttu/dotnet-fake-json-server) - Fake REST API for prototyping or as a CRUD Back End. No need to define types, uses dynamic typing. Data is stored to a single JSON file. Has authentication, WebSocket notifications, async long running operations, random generation for errors/delays and experimental GraphQL support.
 * [gitignore.io](https://github.com/joeblau/gitignore.io) - Create useful .gitignore files for your project [https://www.gitignore.io](https://www.gitignore.io).
-* [GitInfo](https://github.com/kzu/GitInfo) - Git and SemVer Info from MSBuild, C# and VB.
```
Comment (ngohungphuc) [inline README.md]: You remove gitInfo now.Please rebase with master branch

**Pull request closed without merging**